### PR TITLE
Fixup scripts to set up a new CI node

### PR DESCRIPTION
**Pull request by danpaul000**
#### Problem
No consistent way to provision non-Azure CI machines

#### Summary of Changes
Fix scripts used for datacenter/colo/office setup and add a buildkite setup script.  Instructions in `ci/README.md`

### CODE REVIEW

**Comment by danpaul000** [comment]
@mvines this is still a bit of WIP but it's what I've been putting together to more-consistently provision non-cloud imaged machines.

**Comment by danpaul000** [comment]
Also, this whole thing should not be living in `net/...` but I didn't get around to moving it yet.

**Comment by mvines** [comment]
Oh nice, somewhere under `ci/` might be a better home

**Comment by danpaul000** [comment]
@t-nelson @sakridge FYI for setting up any additional physical nodes for CI.

File: ci/README.md [hunk 1]
```diff
@@ -12,9 +12,47 @@ be run on the `default` queue, and the [buildkite artifact
...
+sudo ./setup-new-node/setup-buildkite.sh
+```
+- Copy the pubkey contents from `~buildkite-agent/.ssh/id_ecdsa.pub` and
+add the pubkey as an authorized SSH key on github.
+- Edit `/etc/buildkite-agent/buildkite-agent.cfg` and/or `/etc/systemd/system/buildkite-agent@*` to the desired configuration of the agent(s)
+- Copy `ejson` keys from another CI node at `/opt/ejson/keys/`
+to the same location on the new node.
+- Start the new agent(s) with `sudo systemctl enable --now buildkite-agent`
+
+## Buildkite Azure Setup
```
Comment (mvines) [inline ci/README.md:55]: Can you remove the Azure info actually, we're never going back there

Comment (mvines) [reply ci/README.md:55]: (or move it into the "Reference" section below)

File: ci/setup-new-node/enable-buildkite.sh [hunk 2]
```diff
@@ -0,0 +1,3 @@
+#!/usr/bin/env bash
```
Comment (mvines) [inline ci/setup-new-node/enable-buildkite.sh:1]: "setup-new-node/" -- "node" is an overloaded term.  ""setup-new-buildkite-agent/" ?

**Pull request merged**